gnomad_qc/v4/resources/meta.py

```diff
@@ -31,24 +31,27 @@ def meta_tsv_path(version: str = CURRENT_VERSION, data_type: str = "exomes") ->
 
 
 def meta(
-    version: str = CURRENT_VERSION, data_type: str = "exomes", overwrite: bool = False
+    version: str = CURRENT_VERSION,
+    data_type: str = "exomes",
 ) -> TableResource:
     """
     Get the gnomAD v4 sample QC meta VersionedTableResource.
 
-    :param version: Version of meta that corresponds to its data type.
+    Function will check that metadata exists for the requested gnomAD version and data type
+    and will throw an error if it already exists.
+
+    :param version: gnomAD version.
     :param data_type: Data type ("exomes" or "genomes"). Default is "exomes".
-    :param overwrite: Whether to overwrite existing resources.
     :return: gnomAD v4 sample meta VersionedTableResource.
     """
-    # Check if the meta Table exists
+    # Check if the meta Table exists for specified version and data type combination
     check_resource_existence(
         output_step_resources={
```

Comment:
shouldn't the behavior of this error out if the HT doesn't exist? as in, shouldn't this be `input_step_resources` rather than `output_step_resources`? it should be going through this logic in this function https://github.com/broadinstitute/gnomad_qc/blob/main/gnomad_qc/resource_utils.py#L36

Reply:
Oops, sorry, I misread the usage. I changed it.